src/flow.h

```diff
@@ -373,11 +374,6 @@ typedef struct Flow_
             uint8_t code;   /**< icmp code */
         } icmp_d;
     };
-    uint8_t proto;
```

Comment:
is this just all rearranged for space? There was a grouping to it based on time of access, trying to keep first cache line mostly read only, etc.

Reply:
yes, it was based on space, I'll drop this commit, sorry.